libraries/math/src/processor.rs

```diff
@@ -145,6 +145,15 @@ pub fn process_instruction(
             msg!("{}", result as u64);
             Ok(())
         }
+        MathInstruction::F64Pow { base, exponent } => {
+            msg!("Calculating f64 Pow");
+            sol_log_compute_units();
+            let _result = base.powi(5);
+            let result = base.powf(exponent);
```

Comment:
Is this processor just for testing purposes? I'm curious why you're just sticking 5 in there.

From what I can see in other uses of `spl-math` it's not actually a deployed program?